targets/esp32/mqtt/src/main.c

```diff
@@ -0,0 +1,131 @@
+#include <stdio.h>
+#include <stdint.h>
+#include <stddef.h>
+#include <string.h>
+#include "esp_wifi.h"
+#include "esp_system.h"
+#include "nvs_flash.h"
+#include "esp_event.h"
+#include "esp_netif.h"
+
+#include "freertos/FreeRTOS.h"
+#include "freertos/task.h"
+#include "freertos/semphr.h"
+#include "freertos/queue.h"
+
+#include "lwip/sockets.h"
+#include "lwip/dns.h"
+#include "lwip/netdb.h"
+
+#include "esp_log.h"
+#include "mqtt_client.h"
+#include "cnetwork.h"
+#include "config.h"
+
+#define CLIENT_ID "ESP32"
+
+static const char *TAG = "MQTT_MAINFLUX";
+
+static void log_error_if_nonzero(const char *message, int error_code)
+{
+    if (error_code != 0)
+    {
+        ESP_LOGE(TAG, "Last error %s: 0x%x", message, error_code);
+    }
+}
+void create_mainflux_channel(void)
```

Comment:
Do wee need to create a channel?

Reply:
To create a string that matches the channel string required by mainflux

Reply:
Noted. Should be create topic